code/modules/projectiles/projectile.dm

```diff
@@ -1213,8 +1213,7 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 		return armor_val / total_weight
 
 /mob/living/carbon/xenomorph/get_soft_armor(armor_type, proj_def_zone)
-	return ..() * get_sunder()
-
+	return ..() * get_sunder() + resist_modifiers[armor_type]
```

Comment:
How will this handle resist mods outside of 0-100?

Reply:
Modify by armour handles that, so that's not an issue.